examples/customization/customColumn.html

```diff
@@ -11,8 +11,21 @@
 });
 
 var HeaderComponent = React.createClass({
+  textOnClick: function(e) {
```

Comment:
This is an example in the documentation. Basically we're creating a custom header component to be used as something that will filter the data by the current column.